assets/css/style.css

```diff
@@ -889,6 +889,12 @@ tbody {
 
 /* side list */
 
+.sidebar-scroll {
+    overflow: auto;
+    height: 15vh !important;
```

Comment:
```suggestion
    height: 33vh !important;
```

I propose using `33vh` for the height. It takes up more space, but I think it gives the sidebar more visibility on both display sizes. Screenshots for comparison are below.

Do you agree? Or do you have another view? This is my thought but I am curious to know what you think too.


## Before

![Screenshot of the sidebar with 15vh set as height. A menu in the Meta category is opened on a large display.](https://user-images.githubusercontent.com/4721034/148591909-83480e8e-2f78-4997-9010-a5d28ecad147.png "Screenshot of the sidebar with 15vh set as height. A menu in the Meta category is opened on a large display.")

![Screenshot of the sidebar with 15vh set as height. A menu in the Meta category is opened on a smartphone screen size.](https://user-images.githubusercontent.com/4721034/148592013-7d81962c-2572-4600-b216-fe1bd01cddc4.png "Screenshot of the sidebar with 15vh set as height. A menu in the Meta category is opened on a smartphone screen size.")


## After

![Screenshot of the sidebar with 33vh set as height. A menu in the Meta category is opened on a large display.](https://user-images.githubusercontent.com/4721034/148591588-ab61c79e-fb51-4616-81a6-3f5dc4bb6621.png "Screenshot of the sidebar with 33vh set as height. A menu in the Meta category is opened on a large display.")

![Screenshot of the sidebar with 33vh set as height. A menu in the Meta category is opened on a smartphone screen size.](https://user-images.githubusercontent.com/4721034/148590958-567ac6a2-98cd-4df1-a60e-e51b7dc4ea0e.png "Screenshot of the sidebar with 33vh set as height. A menu in the Meta category is opened on a smartphone screen size.")